all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -23,25 +23,23 @@ items:
 - apiVersion: extensions/v1beta1
   kind: Deployment
   metadata:
-    name: jaeger-all-in-one
+    name: jaeger-deployment
```

Comment:
If we have a need for that in the future, I'm OK in adding that. Adding tags just for the sake of it clutters the UI. If an user really wants to know if it's standalone, the deployment definition can be queried, or it can be checked if other services are available (collector, agent, ...)